src/app/Http/Controllers/PostController.php

```diff
@@ -11,6 +11,12 @@
 
 class PostController extends Controller
 {
+    /** If the rating falls below this treshold, the admin receives a warning. */
+    const WARNING_THRESHOLD_RATING = 2.0;
+
+    /** Minimum number of ratings needed before a warning is send. */
+    const WARNING_THRESHOLD_NUMBER = 1;
```

Comment:
1 warning as threshold is a bit low for reality, great for testing though, but yeah

Reply:
Oh yeah, I put that there when testing. I think I'll keep it there, just to make the demo easier.